src/v1/destinations/am/networkHandler.ts

```diff
@@ -90,7 +80,7 @@ const responseHandler = (responseParams: {
   }
   throw new AbortedError(
     `Request Failed during ${DESTINATION} response transformation: with status "${status}" due to "${
-      JSON.stringify(response.error) || 'unknown error'
+      JSON.stringify(errorResponse) || 'unknown error'
```

Comment:
re-use error Message
```suggestion
      JSON.stringify(errorMessage) || 'unknown error'
```